nncf/torch/utils.py

```diff
@@ -467,3 +470,46 @@ def get_model_dtype(model: torch.nn.Module) -> torch.dtype:
         # The model had no parameters at all, assume FP32
         dtype = torch.float32
     return dtype
+
+
+def get_weight_nodes_in_inference_grpah(
+    inference_nncf_graph: NNCFGraph, mat_mul_metatypes: List[om.PTOperatorMetatype]
```

Comment:
Looks like mat_mul_metatypes expected always same value, mat_mul_metatypes should not be used as argument instead use reusable constant variable

Reply:
Done, please check